subsys/bluetooth/mesh/net.c

```diff
@@ -184,11 +184,13 @@ int bt_mesh_net_create(uint16_t idx, uint8_t flags, const uint8_t key[16],
 	atomic_set_bit_to(bt_mesh.flags, BT_MESH_IVU_IN_PROGRESS,
 			  BT_MESH_IV_UPDATE(flags));
 
-	/* Set minimum required hours, since the 96-hour minimum requirement
-	 * doesn't apply straight after provisioning (since we can't know how
-	 * long has actually passed since the network changed its state).
+	/* If IV Update is already in progress, set minimum required hours,
+	 * since the 96-hour minimum requirement doesn't apply in this case straight
+	 * after provisioning.
```

Comment:
Good catch :+1: Would you mind adding a reference to the specification section for this here (or in the commit message)? Should be 3.11.5

Reply:
I'm confused. The definition in `Mesh spec 1.0.1 section 5 provision 5.4.2.5 distribution of provision data` seems to have another meaning?

`Note: After the processing the Provisioning Data PDU from the Provisioner, the 96 hour time limits for changing the IV Update procedure state, as defined in the IV Update procedure, do not apply.`

Reply:
Yes, this is slightly wrong and underspecified in the published. This should get rectified in the spec eventually.